src/mgr/OSDPerfMetricReport.h

```diff
@@ -0,0 +1,29 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+#ifndef OSD_PERF_METRIC_REPORT_H_
+#define OSD_PERF_METRIC_REPORT_H_
+#include "include/denc.h"
+//#include "mgr/OSDPerfMetricQuery"
+
+typedef int OSDPerfMetricQuery;  //Temporary
+
+struct OSDPerfMetricReport
+{
+  OSDPerfMetricQuery query; 
```

Comment:
No need to include the full query -- if and when we support multiple queues, I think it would be best to match based on a query id assigned by the Mgr and distributed to the OSDs in the ```MMgrConfigure``` message

Reply:
Ok, then i'll move back to including only the ID.

Reply:
Fixed in: https://github.com/ceph/ceph/pull/24215/commits/3a5f9989c9b6402152b475ba295d4b88c2470906